git_machete/tests/functional/test_machete.py

```diff
@@ -36,9 +36,9 @@ def new_repo(self, *args: str) -> "GitRepositorySandbox":
         os.chdir(args[0])
         if len(args) > 1:
             opts = args[1:]
-            self.execute(f"git init {' '.join(opts)}")
+            self.execute(f"git init -q {' '.join(opts)}")
```

Comment:
Nit: the params passed to `execute` are parsed by the shell (i.e. split on spaces into separate words etc., unlike with e.g. `subprocess.Popen` where you need to pass already parsed arguments as list of strings). Thus, you can probably remove the `if len(args) > 1` and just put `' '.join(args[1:])` unconditionally into the string. If `len(args) == 1`, then it'll simply expand to an empty string.